main/guides/getting-started/index.md

```diff
@@ -36,25 +36,22 @@ Currently Agoric supports macOS and Linux (including [Windows Subsystem for Linu
 
 In this section you'll be installing prerequisite components into your environment. If you're working with your own environment rather than using a fresh Ubuntu install, you may already have some or all of these components already installed.
 
-### Installing NVM and Node
+### Installing nvm and Node.js
 
 <details>
-<summary>Installing NVM and Node on Ubuntu</summary>
+<summary>Installing nvm and Node.js on Ubuntu</summary>
 
-At this point the Node Version Manager (NVM) utility will be installed. `nvm` makes it easy to select the specific version of [Node](https://nodejs.org/) that will be required for this tutorial (v18.16.0).
+At this point the [nvm](https://github.com/nvm-sh/nvm) (Node Version Manager) utility will be installed. `nvm` makes it easy to select the specific version of [Node.js](https://nodejs.org/) that will be required for this tutorial (v18.16.0).
 
+First, follow the [nvm installation instructions](https://github.com/nvm-sh/nvm?tab=readme-ov-file#installing-and-updating), including sourcing `nvm.sh` into the shell for later steps.
 ```sh
-wget -O- https://raw.githubusercontent.com/nvm-sh/nvm/v0.39.7/install.sh | bash
+# For example:
+wget -qO- https://raw.githubusercontent.com/nvm-sh/nvm/v0.39.7/install.sh | bash
+export NVM_DIR="$([ -z "${XDG_CONFIG_HOME-}" ] && printf %s "${HOME}/.nvm" || printf %s "${XDG_CONFIG_HOME}/nvm")"
+[ -s "$NVM_DIR/nvm.sh" ] && \. "$NVM_DIR/nvm.sh" # This loads nvm
```

Comment:
Is all this gobbledygook necessary? I'm pretty sure @kbennett2000 tested the `source ~/.bashrc` approach.

Reply:
That only works for bash, while Fred was using zsh. But the most important aspect of this change is just including the https://github.com/nvm-sh/nvm?tab=readme-ov-file#installing-and-updating link; this block is now just an example for one-stop-shopping and could be removed entirely:
<blockquote>

<details open>
<summary>Installing nvm and Node.js on Ubuntu</summary>

At this point the [nvm](https://github.com/nvm-sh/nvm) (Node Version Manager) utility will be installed. `nvm` makes it easy to select the specific version of [Node.js](https://nodejs.org/) that will be required for this tutorial (v18.16.0).

First, follow the [nvm installation instructions](https://github.com/nvm-sh/nvm?tab=readme-ov-file#installing-and-updating), including sourcing `nvm.sh` into the shell for later steps.

Then, install Node.js.
```sh
nvm install v18.18.0
```

</details>

</blockquote>

Reply:
> this block ... could be removed entirely

I think the tutorial is designed to be self-contained. Every command you have to issue is on one page. (Installing Keplr might be one exception where some steps are not covered in full.)

> Fred was using zsh.

The tutorial is _one_ path through the whole process. I think we try to distinguish requirements (node.js) from the detailed existence proof that they can be met (nvm ...).